source/Plugins/ExpressionParser/Swift/SwiftUserExpression.cpp

```diff
@@ -74,6 +74,22 @@ void SwiftUserExpression::DidFinishExecuting() {
   }
 }
 
+static CompilerType getBoundOrUnboundType(CompilerType &valobj_type,
```

Comment:
LLDB API's begin with an initial capitol.  So this should be GetBoundOrUnboundType.  But since those are the only types of generic, why not GetGenericType?

Reply:
Sounds good.